### ISSUE TITLE
tor-browser: install fails due to missing noscript xpi

### ISSUE THREAD

**Issue by kazcw** [NixOS/nixpkgs#64360]
## Issue description
The noscript xpi is failing to download from the mirror (informaction).

### Steps to reproduce
`nix-shell -p tor-browser-bundle`

## Workaround
I don't know if there's a more reliable mirror available, but as a workaround I used nix-prefetch-url to get the file 'noscript-8.1.2.xpi' from some website ( https://legacycollector.org/firefox-addons/722/index.html ) and it passed hash check.

**Comment by kazcw** [comment]
Specifically, the mirror it's trying to fetch from doesn't seem to be available at all:
```
building '/nix/store/sy4741fk5am1cmqcs146c975j7fk15n9-noscript-5.1.2.xpi.drv'...
Switched to a new branch 'fetchgit'
Submodule 'translations' (https://git.torproject.org/translation.git) registered for path 'translations'
Initialized empty Git repository in /nix/store/yx3r1lf912n6xrgrnl7cx8gsllk2w51y-https-everywhere/translations/.git/

trying https://secure.informaction.com/download/releases/noscript-5.1.2.xpi
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (7) Failed to connect to secure.informaction.com port 443: Connection refused
error: cannot download noscript-5.1.2.xpi from any mirror
```

**Comment by FliegendeWurst** [comment]
`tor-browser-bundle` is no longer included in nixpkgs: #77452
